core/src/main/java/com/linecorp/armeria/server/ServerMetrics.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.LongAdder;
+
+/**
+ * A class that holds metrics related server.
+ */
+public final class ServerMetrics {
+
+    private final LongAdder pendingRequests = new LongAdder();
```

Comment:
This doesn't seem to be used

```suggestion
```

Reply:
oh right.. i forgot to delete it after some modifications.. 
i will remove the field!! thank you!!